python-sdk/tests_integration/astro_deploy/master_dag.py

```diff
@@ -0,0 +1,201 @@
+import logging
+import os
+import time
+from datetime import datetime
+from typing import Any, List
+
+from airflow import DAG
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+from airflow.models import DagRun
+from airflow.models.baseoperator import chain
+from airflow.operators.bash import BashOperator
+from airflow.operators.dummy import DummyOperator
+from airflow.operators.python import PythonOperator
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+from airflow.utils.session import create_session
+
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL", "#provider-alert")
+SLACK_WEBHOOK_CONN = os.getenv("SLACK_WEBHOOK_CONN", "http_slack")
+SLACK_USERNAME = os.getenv("SLACK_USERNAME", "airflow_app")
+
+
+def get_report(dag_run_ids: List[str], **context: Any) -> None:
+    """Fetch dags run details and generate report"""
+    with create_session() as session:
+        last_dags_runs: List[DagRun] = session.query(DagRun).filter(DagRun.run_id.in_(dag_run_ids)).all()
+        message_list: List[str] = []
+
+        airflow_version = context["ti"].xcom_pull(task_ids="get_airflow_version")
+        airflow_version_message = f"Airflow version for the below run is `{airflow_version}` \n\n"
+        message_list.append(airflow_version_message)
```

Comment:
Maybe we can add another line at the beginning saying this alert is for Python SDK integration tests since we're sending report to the same provider-alert channel now.

Reply:
rephrase it let me know it looks now